src/app/myworkflows/sidebar-accordion/sidebar-accordion.component.html

```diff
@@ -39,7 +55,15 @@
               </a>
             </mat-list-item>
             <mat-list-item *ngIf="!orgObj || orgObj?.published.length === 0">
-              <p class="text-center size-small">No published workflows.</p>
+              <p data-cy="no-published-apptool-message" *ngIf="pageEntryType === 'apptool'" class="text-center size-small">
```

Comment:
If you change `pageEntryType: string` to `pageEntryType: EntryType`, then change the `ngIf` to `pageEntryType === EntryType.AppTool`, etc., here and the block starting on line 89.